### ISSUE TITLE
Display bug in order creation box

### ISSUE THREAD

**Issue by 27aume** [KomodoPlatform/BarterDEX#223]
I'm not sure if it is intended to do so, when activating market maker option and then switching form buy to sell tab the text in the button change inaccurately.
 
![bdex_interface01](https://user-images.githubusercontent.com/30743589/38951467-1e0afbc6-431f-11e8-8be3-6d357790e54e.jpg)
![bdex_interface02](https://user-images.githubusercontent.com/30743589/38951468-1e386732-431f-11e8-900a-94d7f9a746b9.jpg)

**Comment by satindergrewal** [comment]
yes, in advanced mode, there could possibly be some issues like this UI rendering. thanks for reporting. will have a look soon.
